CREDITS-MUSIC

```diff
@@ -138,14 +138,56 @@ Story: (same as Phase 2)
 	DM32: "Bullet Hell" by rostuhan
 
 
---Previously Featured--
-
-"Intrare In Infernum" by Lola "BlueWorrior" Harvey (E3M1)
-"Panic" by Anthony "Ajanddino" Pierce (E1M6)
-Doom2Alt.mid by Bradley "BobFromReboot" Lavigne (Map12)
-"Fountain Dance" by Tyler "Picklehammer" Pantella
-"I Hate Fredrik Johansson" by Toby "Tobester" Collins Jr.
-"Warpath" by Tyler "Picklehammer" Pantella
-"Mists Over The Mire" by Tyler "Picklehammer" Pantella, modified by Obsidian Plague and m
-"Burn The Bodies" by Tyler "Picklehammer" Pantella
-"Music for Freedoom 10" by Jeremy "Vandalorian" Emerson
+--Previously Featured in 0.12.1--
```

Comment:
The formatting in this section is simpler than the current section above it. That's ok. Older stuff can be a simple flat list since it's less interesting to most readers.